shared/build.gradle.kts

```diff
@@ -12,5 +12,6 @@ dependencies {
 
     listOf(
         libs.bundles.test.kotlin,
+        libs.kotlin.logging,
```

Comment:
now that the grpc bundle isn't bringing in grpc-springboot-starter the tests in this project weren't passing due to a missing logging class... hopefully this fix is reasonable...